modules/hiera/git.tf

```diff
@@ -0,0 +1,9 @@
+module "git" {
```

Comment:
I would merge this back into module `vshn_lb_hieradata` once that's moved to github.com/appuio/terraform-modules.git.

Reply:
Added as a submodule of `lb-vshn-hiera`